llvm/test/Transforms/SCCP/trunc-nuw-nsw-flags.ll

```diff
@@ -0,0 +1,179 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 4
+; RUN: opt -passes=ipsccp -S %s | FileCheck %s
```

Comment:
```suggestion
; RUN: opt -passes=sccp -S %s | FileCheck %s
```

Reply:
Oh, sorry for merging this patch without noticing this suggestion. Changing ipsccp to sccp would cause test `range_from_sge_sle` to lose `nuw` flag. I think the test coverage of ipsccp is needed here.

Reply:
Oh I missed this.

Reply:
We usually use IPSCCP in tests because it uses PredicateInfo, while plain SCCP currently doesn't.